### DISCUSSION TITLE
Support for CUDA 113 GTX880M Kepler core :(

### DISCUSSION THREAD

**Discussion by Frostfire83** [nagadomi/nunif#280] [General]
Hello,

I am trying to get support for my card GTX 880M I got as far as installing latest drivers with CUDA 11.4 and tried running installer.bat while changing the requirements for pytorch 1.12.1+cu113 and torchvision 0.13.1+cu113. but im having an error and I need help...
I know this is a very old card but its all I got... I enjoy the tool greatly but im only able to run it on CPU and it takes too much time to render anything... :(

Below is the output:

Install Python...                                                                                                       Successfully installed Python                                                                                           Install pip...                                                                                                          Collecting pip
  Using cached pip-24.3.1-py3-none-any.whl.metadata (3.7 kB)
Collecting setuptools
  Using cached setuptools-75.6.0-py3-none-any.whl.metadata (6.7 kB)
Collecting wheel
  Using cached wheel-0.45.1-py3-none-any.whl.metadata (2.3 kB)
Using cached pip-24.3.1-py3-none-any.whl (1.8 MB)
Using cached setuptools-75.6.0-py3-none-any.whl (1.2 MB)
Using cached wheel-0.45.1-py3-none-any.whl (72 kB)
Installing collected packages: wheel, setuptools, pip
Successfully installed pip-24.3.1 setuptools-75.6.0 wheel-0.45.1
Successfully installed pip
Cloning into 'C:\Users\Newadmin\Downloads\nunif-windows\nunif'...
remote: Enumerating objects: 10198, done.
remote: Counting objects: 100% (1951/1951), done.
remote: Compressing objects: 100% (232/232), done.
remote: Total 10198 (delta 1744), reused 1745 (delta 1719), pack-reused 8247 (from 2)
Receiving objects: 100% (10198/10198), 9.11 MiB | 1.94 MiB/s, done.
Resolving deltas: 100% (7200/7200), done.
Install Python Packages...
Requirement already satisfied: pip in c:\users\newadmin\downloads\nunif-windows\python\lib\site-packages (24.3.1)
Looking in indexes: https://pypi.org/simple, https://download.pytorch.org/whl/cpu, https://download.pytorch.org/whl/cu113
Ignoring torch: markers 'sys_platform == "darwin"' don't match your environment
Ignoring torchvision: markers 'sys_platform == "darwin"' don't match your environment
Ignoring torch: markers 'sys_platform == "linux"' don't match your environment
Ignoring torchvision: markers 'sys_platform == "linux"' don't match your environment
Collecting torch==1.12.1+cu113 (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements-torch.txt (line 13))
  Downloading https://download.pytorch.org/whl/cu113/torch-1.12.1%2Bcu113-cp310-cp310-win_amd64.whl (2143.8 MB)
     ---------------------------------------- 2.1/2.1 GB 11.9 MB/s eta 0:00:00
Collecting torchvision==0.13.1+cu113 (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements-torch.txt (line 14))
  Downloading https://download.pytorch.org/whl/cu113/torchvision-0.13.1%2Bcu113-cp310-cp310-win_amd64.whl (4.7 MB)
     ---------------------------------------- 4.7/4.7 MB 11.9 MB/s eta 0:00:00
Collecting typing-extensions (from torch==1.12.1+cu113->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements-torch.txt (line 13))
  Downloading typing_extensions-4.12.2-py3-none-any.whl.metadata (3.0 kB)
Collecting numpy (from torchvision==0.13.1+cu113->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements-torch.txt (line 14))
  Downloading numpy-2.2.1-cp310-cp310-win_amd64.whl.metadata (60 kB)
Collecting requests (from torchvision==0.13.1+cu113->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements-torch.txt (line 14))
  Downloading requests-2.32.3-py3-none-any.whl.metadata (4.6 kB)
Collecting pillow!=8.3.*,>=5.3.0 (from torchvision==0.13.1+cu113->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements-torch.txt (line 14))
  Downloading pillow-11.0.0-cp310-cp310-win_amd64.whl.metadata (9.3 kB)
Collecting charset-normalizer<4,>=2 (from requests->torchvision==0.13.1+cu113->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements-torch.txt (line 14))
  Downloading charset_normalizer-3.4.1-cp310-cp310-win_amd64.whl.metadata (36 kB)
Collecting idna<4,>=2.5 (from requests->torchvision==0.13.1+cu113->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements-torch.txt (line 14))
  Downloading idna-3.10-py3-none-any.whl.metadata (10 kB)
Collecting urllib3<3,>=1.21.1 (from requests->torchvision==0.13.1+cu113->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements-torch.txt (line 14))
  Downloading urllib3-2.3.0-py3-none-any.whl.metadata (6.5 kB)
Collecting certifi>=2017.4.17 (from requests->torchvision==0.13.1+cu113->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements-torch.txt (line 14))
  Downloading certifi-2024.12.14-py3-none-any.whl.metadata (2.3 kB)
Downloading pillow-11.0.0-cp310-cp310-win_amd64.whl (2.6 MB)
   ---------------------------------------- 2.6/2.6 MB 13.4 MB/s eta 0:00:00
Downloading numpy-2.2.1-cp310-cp310-win_amd64.whl (12.9 MB)
   ---------------------------------------- 12.9/12.9 MB 10.9 MB/s eta 0:00:00
Downloading requests-2.32.3-py3-none-any.whl (64 kB)
Downloading typing_extensions-4.12.2-py3-none-any.whl (37 kB)
Downloading certifi-2024.12.14-py3-none-any.whl (164 kB)
Downloading charset_normalizer-3.4.1-cp310-cp310-win_amd64.whl (102 kB)
Downloading idna-3.10-py3-none-any.whl (70 kB)
Downloading urllib3-2.3.0-py3-none-any.whl (128 kB)
Installing collected packages: urllib3, typing-extensions, pillow, numpy, idna, charset-normalizer, certifi, torch, requests, torchvision
Successfully installed certifi-2024.12.14 charset-normalizer-3.4.1 idna-3.10 numpy-2.2.1 pillow-11.0.0 requests-2.32.3 torch-1.12.1+cu113 torchvision-0.13.1+cu113 typing-extensions-4.12.2 urllib3-2.3.0
Collecting tqdm (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 7))
  Downloading tqdm-4.67.1-py3-none-any.whl.metadata (57 kB)
Collecting wand>=0.6.10 (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 8))
  Downloading Wand-0.6.13-py2.py3-none-any.whl.metadata (4.0 kB)
Collecting fonttools (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 9))
  Downloading fonttools-4.55.3-cp310-cp310-win_amd64.whl.metadata (168 kB)
Collecting scipy (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 10))
  Downloading scipy-1.14.1-cp310-cp310-win_amd64.whl.metadata (60 kB)
Collecting waitress (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 11))
  Downloading waitress-3.0.2-py3-none-any.whl.metadata (5.8 kB)
Collecting bottle (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 12))
  Downloading bottle-0.13.2-py2.py3-none-any.whl.metadata (1.8 kB)
Collecting diskcache (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 13))
  Downloading diskcache-5.6.3-py3-none-any.whl.metadata (20 kB)
Collecting flake8 (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 14))
  Downloading flake8-7.1.1-py2.py3-none-any.whl.metadata (3.8 kB)
Collecting psutil (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 15))
  Downloading psutil-6.1.1-cp37-abi3-win_amd64.whl.metadata (23 kB)
Collecting pyyaml (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 16))
  Downloading PyYAML-6.0.2-cp310-cp310-win_amd64.whl.metadata (2.1 kB)
Collecting onnx (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 17))
  Downloading onnx-1.17.0-cp310-cp310-win_amd64.whl.metadata (16 kB)
Collecting onnxconverter_common (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 18))
  Downloading onnxconverter_common-1.14.0-py2.py3-none-any.whl.metadata (4.2 kB)
Collecting lpips (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 19))
  Downloading lpips-0.1.4-py3-none-any.whl.metadata (10 kB)
Collecting packaging (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 21))
  Downloading packaging-24.2-py3-none-any.whl.metadata (3.2 kB)
Collecting dctorch (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 24))
  Downloading dctorch-0.1.2-py3-none-any.whl.metadata (607 bytes)
Collecting timm (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 27))
  Downloading timm-1.0.12-py3-none-any.whl.metadata (51 kB)
Collecting numba (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 28))
  Downloading numba-0.60.0-cp310-cp310-win_amd64.whl.metadata (2.8 kB)
Collecting av<14.0.0,>=12.2.0 (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 29))
  Downloading av-13.1.0-cp310-cp310-win_amd64.whl.metadata (4.6 kB)
Collecting rembg (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading rembg-2.0.61-py3-none-any.whl.metadata (18 kB)
Collecting truststore (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 31))
  Downloading truststore-0.10.0-py3-none-any.whl.metadata (4.4 kB)
Collecting colorama (from tqdm->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 7))
  Downloading colorama-0.4.6-py2.py3-none-any.whl.metadata (17 kB)
Requirement already satisfied: numpy<2.3,>=1.23.5 in c:\users\newadmin\downloads\nunif-windows\python\lib\site-packages (from scipy->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 10)) (2.2.1)
Collecting mccabe<0.8.0,>=0.7.0 (from flake8->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 14))
  Downloading mccabe-0.7.0-py2.py3-none-any.whl.metadata (5.0 kB)
Collecting pycodestyle<2.13.0,>=2.12.0 (from flake8->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 14))
  Downloading pycodestyle-2.12.1-py2.py3-none-any.whl.metadata (4.5 kB)
Collecting pyflakes<3.3.0,>=3.2.0 (from flake8->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 14))
  Downloading pyflakes-3.2.0-py2.py3-none-any.whl.metadata (3.5 kB)
Collecting protobuf>=3.20.2 (from onnx->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 17))
  Downloading protobuf-5.29.2-cp310-abi3-win_amd64.whl.metadata (592 bytes)
  Downloading protobuf-3.20.2-cp310-cp310-win_amd64.whl.metadata (698 bytes)
Requirement already satisfied: torch>=0.4.0 in c:\users\newadmin\downloads\nunif-windows\python\lib\site-packages (from lpips->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 19)) (1.12.1+cu113)
Requirement already satisfied: torchvision>=0.2.1 in c:\users\newadmin\downloads\nunif-windows\python\lib\site-packages (from lpips->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 19)) (0.13.1+cu113)
Collecting numpy<2.3,>=1.23.5 (from scipy->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 10))
  Downloading numpy-1.26.4-cp310-cp310-win_amd64.whl.metadata (61 kB)
Collecting huggingface_hub (from timm->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 27))
  Downloading huggingface_hub-0.27.0-py3-none-any.whl.metadata (13 kB)
Collecting safetensors (from timm->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 27))
  Downloading safetensors-0.4.5-cp310-none-win_amd64.whl.metadata (3.9 kB)
Collecting llvmlite<0.44,>=0.43.0dev0 (from numba->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 28))
  Downloading llvmlite-0.43.0-cp310-cp310-win_amd64.whl.metadata (4.9 kB)
Collecting jsonschema (from rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading jsonschema-4.23.0-py3-none-any.whl.metadata (7.9 kB)
Collecting opencv-python-headless (from rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading opencv_python_headless-4.10.0.84-cp37-abi3-win_amd64.whl.metadata (20 kB)
Requirement already satisfied: pillow in c:\users\newadmin\downloads\nunif-windows\python\lib\site-packages (from rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30)) (11.0.0)
Collecting pooch (from rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading pooch-1.8.2-py3-none-any.whl.metadata (10 kB)
Collecting pymatting (from rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading PyMatting-1.1.13-py3-none-any.whl.metadata (7.5 kB)
Collecting scikit-image (from rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading scikit_image-0.25.0-cp310-cp310-win_amd64.whl.metadata (14 kB)
Requirement already satisfied: typing-extensions in c:\users\newadmin\downloads\nunif-windows\python\lib\site-packages (from torch>=0.4.0->lpips->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 19)) (4.12.2)
Requirement already satisfied: requests in c:\users\newadmin\downloads\nunif-windows\python\lib\site-packages (from torchvision>=0.2.1->lpips->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 19)) (2.32.3)
Collecting filelock (from huggingface_hub->timm->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 27))
  Downloading filelock-3.16.1-py3-none-any.whl.metadata (2.9 kB)
Collecting fsspec>=2023.5.0 (from huggingface_hub->timm->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 27))
  Downloading fsspec-2024.12.0-py3-none-any.whl.metadata (11 kB)
Collecting attrs>=22.2.0 (from jsonschema->rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading attrs-24.3.0-py3-none-any.whl.metadata (11 kB)
Collecting jsonschema-specifications>=2023.03.6 (from jsonschema->rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading jsonschema_specifications-2024.10.1-py3-none-any.whl.metadata (3.0 kB)
Collecting referencing>=0.28.4 (from jsonschema->rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading referencing-0.35.1-py3-none-any.whl.metadata (2.8 kB)
Collecting rpds-py>=0.7.1 (from jsonschema->rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading rpds_py-0.22.3-cp310-cp310-win_amd64.whl.metadata (4.2 kB)
Collecting platformdirs>=2.5.0 (from pooch->rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading platformdirs-4.3.6-py3-none-any.whl.metadata (11 kB)
Collecting networkx>=3.0 (from scikit-image->rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading networkx-3.4.2-py3-none-any.whl.metadata (6.3 kB)
Collecting imageio!=2.35.0,>=2.33 (from scikit-image->rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading imageio-2.36.1-py3-none-any.whl.metadata (5.2 kB)
Collecting tifffile>=2022.8.12 (from scikit-image->rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading tifffile-2024.12.12-py3-none-any.whl.metadata (31 kB)
Collecting lazy-loader>=0.4 (from scikit-image->rembg->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 30))
  Downloading lazy_loader-0.4-py3-none-any.whl.metadata (7.6 kB)
Requirement already satisfied: charset-normalizer<4,>=2 in c:\users\newadmin\downloads\nunif-windows\python\lib\site-packages (from requests->torchvision>=0.2.1->lpips->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 19)) (3.4.1)
Requirement already satisfied: idna<4,>=2.5 in c:\users\newadmin\downloads\nunif-windows\python\lib\site-packages (from requests->torchvision>=0.2.1->lpips->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 19)) (3.10)
Requirement already satisfied: urllib3<3,>=1.21.1 in c:\users\newadmin\downloads\nunif-windows\python\lib\site-packages (from requests->torchvision>=0.2.1->lpips->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 19)) (2.3.0)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\newadmin\downloads\nunif-windows\python\lib\site-packages (from requests->torchvision>=0.2.1->lpips->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements.txt (line 19)) (2024.12.14)
Downloading tqdm-4.67.1-py3-none-any.whl (78 kB)
Downloading Wand-0.6.13-py2.py3-none-any.whl (143 kB)
Downloading fonttools-4.55.3-cp310-cp310-win_amd64.whl (2.2 MB)
   ---------------------------------------- 2.2/2.2 MB 13.9 MB/s eta 0:00:00
Downloading scipy-1.14.1-cp310-cp310-win_amd64.whl (44.8 MB)
   ---------------------------------------- 44.8/44.8 MB 8.7 MB/s eta 0:00:00
Downloading waitress-3.0.2-py3-none-any.whl (56 kB)
Downloading bottle-0.13.2-py2.py3-none-any.whl (104 kB)
Downloading diskcache-5.6.3-py3-none-any.whl (45 kB)
Downloading flake8-7.1.1-py2.py3-none-any.whl (57 kB)
Downloading psutil-6.1.1-cp37-abi3-win_amd64.whl (254 kB)
Downloading PyYAML-6.0.2-cp310-cp310-win_amd64.whl (161 kB)
Downloading onnx-1.17.0-cp310-cp310-win_amd64.whl (14.5 MB)
   ---------------------------------------- 14.5/14.5 MB 8.2 MB/s eta 0:00:00
Downloading onnxconverter_common-1.14.0-py2.py3-none-any.whl (84 kB)
Downloading protobuf-3.20.2-cp310-cp310-win_amd64.whl (904 kB)
   ---------------------------------------- 904.0/904.0 kB 8.2 MB/s eta 0:00:00
Downloading lpips-0.1.4-py3-none-any.whl (53 kB)
Downloading packaging-24.2-py3-none-any.whl (65 kB)
Downloading dctorch-0.1.2-py3-none-any.whl (2.3 kB)
Downloading timm-1.0.12-py3-none-any.whl (2.4 MB)
   ---------------------------------------- 2.4/2.4 MB 7.0 MB/s eta 0:00:00
Downloading numba-0.60.0-cp310-cp310-win_amd64.whl (2.7 MB)
   ---------------------------------------- 2.7/2.7 MB 6.5 MB/s eta 0:00:00
Downloading av-13.1.0-cp310-cp310-win_amd64.whl (25.8 MB)
   ---------------------------------------- 25.8/25.8 MB 7.5 MB/s eta 0:00:00
Downloading rembg-2.0.61-py3-none-any.whl (40 kB)
Downloading truststore-0.10.0-py3-none-any.whl (18 kB)
Downloading llvmlite-0.43.0-cp310-cp310-win_amd64.whl (28.1 MB)
   ---------------------------------------- 28.1/28.1 MB 6.3 MB/s eta 0:00:00
Downloading mccabe-0.7.0-py2.py3-none-any.whl (7.3 kB)
Downloading numpy-1.26.4-cp310-cp310-win_amd64.whl (15.8 MB)
   ---------------------------------------- 15.8/15.8 MB 7.3 MB/s eta 0:00:00
Downloading pycodestyle-2.12.1-py2.py3-none-any.whl (31 kB)
Downloading pyflakes-3.2.0-py2.py3-none-any.whl (62 kB)
Downloading colorama-0.4.6-py2.py3-none-any.whl (25 kB)
Downloading huggingface_hub-0.27.0-py3-none-any.whl (450 kB)
Downloading jsonschema-4.23.0-py3-none-any.whl (88 kB)
Downloading opencv_python_headless-4.10.0.84-cp37-abi3-win_amd64.whl (38.8 MB)
   ---------------------------------------- 38.8/38.8 MB 7.3 MB/s eta 0:00:00
Downloading pooch-1.8.2-py3-none-any.whl (64 kB)
Downloading PyMatting-1.1.13-py3-none-any.whl (54 kB)
Downloading safetensors-0.4.5-cp310-none-win_amd64.whl (285 kB)
Downloading scikit_image-0.25.0-cp310-cp310-win_amd64.whl (12.8 MB)
   ---------------------------------------- 12.8/12.8 MB 7.2 MB/s eta 0:00:00
Downloading attrs-24.3.0-py3-none-any.whl (63 kB)
Downloading fsspec-2024.12.0-py3-none-any.whl (183 kB)
Downloading imageio-2.36.1-py3-none-any.whl (315 kB)
Downloading jsonschema_specifications-2024.10.1-py3-none-any.whl (18 kB)
Downloading lazy_loader-0.4-py3-none-any.whl (12 kB)
Downloading networkx-3.4.2-py3-none-any.whl (1.7 MB)
   ---------------------------------------- 1.7/1.7 MB 7.2 MB/s eta 0:00:00
Downloading platformdirs-4.3.6-py3-none-any.whl (18 kB)
Downloading referencing-0.35.1-py3-none-any.whl (26 kB)
Downloading rpds_py-0.22.3-cp310-cp310-win_amd64.whl (231 kB)
Downloading tifffile-2024.12.12-py3-none-any.whl (227 kB)
Downloading filelock-3.16.1-py3-none-any.whl (16 kB)
Installing collected packages: wand, bottle, waitress, truststore, safetensors, rpds-py, pyyaml, pyflakes, pycodestyle, psutil, protobuf, platformdirs, packaging, numpy, networkx, mccabe, llvmlite, fsspec, fonttools, filelock, diskcache, colorama, av, attrs, tqdm, tifffile, scipy, referencing, pooch, opencv-python-headless, onnx, numba, lazy-loader, imageio, flake8, scikit-image, pymatting, onnxconverter_common, lpips, jsonschema-specifications, huggingface_hub, dctorch, timm, jsonschema, rembg
  Attempting uninstall: numpy
    Found existing installation: numpy 2.2.1
    Uninstalling numpy-2.2.1:
      Successfully uninstalled numpy-2.2.1
Successfully installed attrs-24.3.0 av-13.1.0 bottle-0.13.2 colorama-0.4.6 dctorch-0.1.2 diskcache-5.6.3 filelock-3.16.1 flake8-7.1.1 fonttools-4.55.3 fsspec-2024.12.0 huggingface_hub-0.27.0 imageio-2.36.1 jsonschema-4.23.0 jsonschema-specifications-2024.10.1 lazy-loader-0.4 llvmlite-0.43.0 lpips-0.1.4 mccabe-0.7.0 networkx-3.4.2 numba-0.60.0 numpy-1.26.4 onnx-1.17.0 onnxconverter_common-1.14.0 opencv-python-headless-4.10.0.84 packaging-24.2 platformdirs-4.3.6 pooch-1.8.2 protobuf-3.20.2 psutil-6.1.1 pycodestyle-2.12.1 pyflakes-3.2.0 pymatting-1.1.13 pyyaml-6.0.2 referencing-0.35.1 rembg-2.0.61 rpds-py-0.22.3 safetensors-0.4.5 scikit-image-0.25.0 scipy-1.14.1 tifffile-2024.12.12 timm-1.0.12 tqdm-4.67.1 truststore-0.10.0 waitress-3.0.2 wand-0.6.13
Collecting wxpython>=4.0.0 (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements-gui.txt (line 1))
  Downloading wxPython-4.2.2-cp310-cp310-win_amd64.whl.metadata (3.1 kB)
Collecting pywin32 (from -r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements-gui.txt (line 2))
  Downloading pywin32-308-cp310-cp310-win_amd64.whl.metadata (8.3 kB)
Collecting six (from wxpython>=4.0.0->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements-gui.txt (line 1))
  Downloading six-1.17.0-py2.py3-none-any.whl.metadata (1.7 kB)
Requirement already satisfied: numpy in c:\users\newadmin\downloads\nunif-windows\python\lib\site-packages (from wxpython>=4.0.0->-r C:\Users\Newadmin\Downloads\nunif-windows\nunif\requirements-gui.txt (line 1)) (1.26.4)
Downloading wxPython-4.2.2-cp310-cp310-win_amd64.whl (16.7 MB)
   ---------------------------------------- 16.7/16.7 MB 10.7 MB/s eta 0:00:00
Downloading pywin32-308-cp310-cp310-win_amd64.whl (6.6 MB)
   ---------------------------------------- 6.6/6.6 MB 12.6 MB/s eta 0:00:00
Downloading six-1.17.0-py2.py3-none-any.whl (11 kB)
Installing collected packages: pywin32, six, wxpython
Successfully installed pywin32-308 six-1.17.0 wxpython-4.2.2
Download Models...
Waifu2x Models: 100%|███████████████████████| 438M/438M [00:41<00:00, 10.6MiB/s]
Downloading: "https://github.com/nagadomi/MiDaS_iw3/zipball/master" to C:\Users\Newadmin\Downloads\nunif-windows\nunif\iw3\pretrained_models\hub\master.zip
Downloading: "https://github.com/nagadomi/ZoeDepth_iw3/zipball/main" to C:\Users\Newadmin\Downloads\nunif-windows\nunif\iw3\pretrained_models\hub\main.zip
Downloading: "https://github.com/nagadomi/Depth-Anything_iw3/zipball/main" to C:\Users\Newadmin\Downloads\nunif-windows\nunif\iw3\pretrained_models\hub\main.zip
Downloading: "https://github.com/nagadomi/Depth-Anything_iw3/zipball/main" to C:\Users\Newadmin\Downloads\nunif-windows\nunif\iw3\pretrained_models\hub\main.zip
Traceback (most recent call last):
  File "runpy.py", line 196, in _run_module_as_main
  File "runpy.py", line 86, in _run_code
  File "C:\Users\Newadmin\Downloads\nunif-windows\nunif\iw3\download_models.py", line 13, in <module>
    main()
  File "C:\Users\Newadmin\Downloads\nunif-windows\nunif\iw3\download_models.py", line 9, in main
    ZU.load_model(gpu=-1)
  File "C:\Users\Newadmin\Downloads\nunif-windows\nunif\iw3\zoedepth_model.py", line 36, in load_model
    model = torch.hub.load("nagadomi/ZoeDepth_iw3:main", model_type, config_mode="infer",
  File "C:\Users\Newadmin\Downloads\nunif-windows\python\lib\site-packages\torch\hub.py", line 540, in load
    model = _load_local(repo_or_dir, model, *args, **kwargs)
  File "C:\Users\Newadmin\Downloads\nunif-windows\python\lib\site-packages\torch\hub.py", line 569, in _load_local
    model = entry(*args, **kwargs)
  File "C:\Users\Newadmin\Downloads\nunif-windows\nunif\iw3\pretrained_models\hub\nagadomi_ZoeDepth_iw3_main\hubconf.py", line 69, in ZoeD_N
    model = build_model(config)
  File "C:\Users\Newadmin\Downloads\nunif-windows\nunif\iw3\pretrained_models\hub\nagadomi_ZoeDepth_iw3_main\zoedepth\models\builder.py", line 51, in build_model
    return get_version(config.version_name).build_from_config(config)
  File "C:\Users\Newadmin\Downloads\nunif-windows\nunif\iw3\pretrained_models\hub\nagadomi_ZoeDepth_iw3_main\zoedepth\models\zoedepth\zoedepth_v1.py", line 252, in build_from_config
    return ZoeDepth.build(**config)
  File "C:\Users\Newadmin\Downloads\nunif-windows\nunif\iw3\pretrained_models\hub\nagadomi_ZoeDepth_iw3_main\zoedepth\models\zoedepth\zoedepth_v1.py", line 247, in build
    model = load_state_from_resource(model, pretrained_resource)
  File "C:\Users\Newadmin\Downloads\nunif-windows\nunif\iw3\pretrained_models\hub\nagadomi_ZoeDepth_iw3_main\zoedepth\models\model_io.py", line 94, in load_state_from_resource
    return load_state_dict_from_url(model, url, progress=True)
  File "C:\Users\Newadmin\Downloads\nunif-windows\nunif\iw3\pretrained_models\hub\nagadomi_ZoeDepth_iw3_main\zoedepth\models\model_io.py", line 70, in load_state_dict_from_url
    state_dict = torch.hub.load_state_dict_from_url(url, map_location='cpu', weights_only=True, **kwargs)
TypeError: load_state_dict_from_url() got an unexpected keyword argument 'weights_only'
Error
Press any key to continue . . .

**Comment by nagadomi** [comment]
pytorch 1.x is not supported.
For iw3, there is no possibility to support pytorch 1.x because it uses many features that are only available in pytorch 2.0 or later.
For waifu2x, it can support pytorch 1.x.

**Reply by Frostfire83** [reply]
Thank you very much for the fast reply! so anything past 2.0 got it. unfortunately that means im most likely SOL. I reached my CUDA driver limit with a modded 475.14 driver with cuda 11.4... I even started reverse engineering some of the .py files with chatGPT lol but if you're telling there's nothing I can do then I guess there's nothing I can do... :(
